Resources/Prototypes/_StarLight/Entities/Objects/Weapons/Guns/LMGs/lmgs.yml

```
@@ -0,0 +1,42 @@
+    - state: mag-3
+      map: ["enum.GunVisualLayers.Mag"]
+  - type: MagazineVisuals
+    magState: mag
+    steps: 4
+    zeroVisible: true
+  - type: Gun
+    minAngle: 20
+    maxAngle: 35
+    angleIncrease: 2
+    angleDecay: 10
+    fireRate: 6  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)